src/App.tsx

```diff
@@ -1,61 +1,114 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { UserList } from './components/UserList/UserList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
```

Comment:
Consider renaming the function to `findUserById` for better readability and clarity. 🤓